src/modules/pages/components/RouteLayouts/UserNavigation.css

```diff
@@ -138,7 +138,12 @@
 }
 
 .buttonsWrapper {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
   margin-right: 12px;
+  padding: 1px;
```

Comment:
Can you please contact Karol & confirm with him if it should be 1px or 2px. I think that it's 1 px but just to make sure.

Reply:
Thank you, I have reached out to confirm. I have added a PR in the meantime, which should fix slight misalignment of text.

Reply:
Thank you, confirmed with Karol that it is 1px.